editor/filesystem_dock.cpp

```diff
@@ -1427,18 +1427,18 @@ void FileSystemDock::_make_dir_confirm() {
 		directory = directory.get_base_dir();
 	}
 
-	if (EditorFileSystem::get_singleton()->get_filesystem_path(directory + dir_name)) {
-		EditorNode::get_singleton()->show_warning(TTR("Could not create folder. File with that name already exists."));
-		return;
-	}
-
 	print_verbose("Making folder " + dir_name + " in " + directory);
 	DirAccessRef da = DirAccess::create(DirAccess::ACCESS_RESOURCES);
 	Error err = da->change_dir(directory);
-	if (err == OK) {
-		err = da->make_dir(dir_name);
+	ERR_FAIL_COND_MSG(!err, "Cannot open directory '" + directory + "'.");
```

Comment:
The condition seems backwars, `OK` is 0 so this will fail when it succeeds.
It should fail on `err`, or more explicit, `err != OK`.

Reply:
Yep yep, my bad